### PR TITLE
Fix countries with apostrophes

### PR DESCRIPTION
**Pull request by snowblink**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -33,14 +33,14 @@ end
...
-  `mkdir output -p`
+  `mkdir -p output`
   data = I18nData.countries(language.upcase)
   File.open("output/countries_#{language.downcase}.yml",'w') {|f|f.puts data.to_yaml}
 end
 
 desc "write example output, just to show off :D"
 task :example_output do
-  `mkdir example_output -p`
+  `mkdir -p example_output`
```
Comment (grosser) [inline Rakefile]: yeah I like that better too, did that not work on any platform or is it just cleanup ?

File: lib/i18n_data/live_data_provider.rb [hunk 2]
```diff
@@ -80,7 +80,7 @@ def english_languages
       @english_languages ||= begin
         codes = {}
         xml(:languages).elements.each('*/iso_639_entry') do |entry|
-          name = entry.attributes['name'].to_s.gsub("'", "\\'")
```
Comment (grosser) [inline lib/i18n_data/live_data_provider.rb]: wonder why I added that ... damn ...

**Comment by grosser** [comment]
thx for digging into this weirdness :D

**Comment by grosser** [comment]
0.6.3 is out! :)

File: Rakefile (hunk 1 above)
Comment (snowblink) [inline Rakefile]: I created directories called "-p" on a mac!

**Comment by snowblink** [comment]
Thanks!

**Pull request merged**